index.html

```diff
@@ -302,17 +301,22 @@ <h1 class="contat">Contact me</h1>
                             a project that needs coding. I’d love to help with it </p>
                     </div>
 
-                    <div class="form">
-                        <input type="text" id="fname" class="form1" name="firstname" placeholder="Yeremias">
-                        <input type="text" id="lname" name="lastname" placeholder="natajayanj14@gmail.com">
-                        <textarea id="subject" name="subject" class="form3" placeholder="Write your message here"
-                            style="height:90px"></textarea>
-
-                    </div>
+                    <form class="form1" action="https://formspree.io/f/xgeqypdd" method="post">
+                        <ul class="widgets">
+                            <li><input type="text" id="fname" class="form1" name="firstname" maxlength="30"
+                                    placeholder="Abdimulhin" required></li>
+                            <li><input type="text" id="lname" name="lastname" placeholder="natajayanj14@gmail.com"
+                                    required></li>
```

Comment:
- [ ] Please replace the type from **text** to **email** of your email input field in line 308 to make this input field follow the email validation :+1: